crates/solver/src/liquidity/order_converter.rs

```diff
@@ -69,44 +72,12 @@ impl OrderConverter {
 
         // The reported fee amount that is used for objective computation is the
         // order's full full amount scaled by a constant factor.
-        let mut solver_fee = remaining.remaining(order.metadata.solver_fee)?;
-        let mut sell_amount = remaining.remaining(sell_amount)?;
-        let mut buy_amount = remaining.remaining(order.data.buy_amount)?;
-
-        // Partially fillable orders are included in the auction when there is at least
-        // 1 atom balance available.
-        if order.data.partially_fillable {
```

Comment:
This logic was moved to `shared::remaining_amounts` module so it can be shared here and with `autopilot::solvable_orders` module.